Catalog/README.md

```diff
@@ -0,0 +1,53 @@
+# Low Code App Editor
+
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=SkylineCommunications_Low-Code-App-Extensions&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=SkylineCommunications_Low-Code-App-Extensions)
+
+The **Low Code App Editor** simplifies managing low-code applications within the DataMiner System. It allows you to import, export, edit, and delete apps while offering customization options for metadata, resources, and deployment settings. With features like tailored export configurations and cross-agent synchronization, the tool streamlines app lifecycle tasks, making it a practical solution for maintaining and deploying low-code apps efficiently.
+
+> [!IMPORTANT]
+> After the deploy you'll need to manually replace on of the references.
+>
+> C:\Skyline DataMiner\ProtocolScripts\WebApiLib.dll needs to be C:\Skyline DataMiner\Webpages\API\bin\WebApiLib.dll
+
+![Overview](Images/LCAEditor_1_0_0_13.gif)
+
+## Installation
+
+#### Step 1: Deploy the Low Code App Editor package
+
+1. Click the **Deploy** button to deploy the package directly to your DataMiner System.
+1. Optionally, go to [admin.dataminer.services](https://admin.dataminer.services/) and verify whether the deployment was successfull.
+
+#### Step 2: Update the WebApiLib reference
+
+1. Connect to your DataMiner system using Cube.
+1. Navigate to the Automation module and search for **Low Code App Editor** automation script.
+1. Scroll down to the bottom and open the **Advanced** dropdown menu.
+1. In the **DLL reference** text box replace the following line *C:\Skyline DataMiner\ProtocolScripts\WebApiLib.dll* with *C:\Skyline DataMiner\Webpages\API\bin\WebApiLib.dll*
```

Comment:
Is this applicable for all DM versions? Is there any resolution in the future?

Reply:
Yes, I think for all supported version at least. Previously, it was possible to link to other DLLs, but Data Acq removed this feature upon realizing it was only being used for the SRM DLL. As a result, they restricted references to the ProtocolScripts folder, with a hardcoded exception specifically for the SRM DLL.